Sources/Core/Views/ProfileImageView/ProfileImageView.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  ProfileImageView.swift
+//  PovioKit
+//
+//  Created by Arlind Dushi on 3/9/22.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 13, *)
+public class ProfileImageProperties: ObservableObject {
+  @Published public var placeHolder = Image(systemName: "person")
+  @Published public var backgroundType = ProfileImageView.BackgroundType.plain(.clear)
+  @Published public var cornerRadius: CGFloat = 0
+  @Published public var contentMode: ContentMode = .fit
+  @Published public var borderColor: Color = .clear
+  @Published public var borderWidth: CGFloat = 0
+  @Published public var badging: ProfileImageView.BadgingMode = .none
+  @Published public var badgeAlignment: Alignment = .bottomTrailing
```

Comment:
Interesting approach. I would prefer going with one reference e.g. `Badge` where you can define all values, alignment as well 🤷‍♂️ 
What do you think?